### PR TITLE
Update financing and add payment milestone type

### LINKED ISSUE #937: milestoneType: Clarify 'financing' code description
As described in https://github.com/open-contracting/standard/issues/438#issuecomment-554032553, the code is ambiguous, and can be interpreted as being appropriate for bill payments. The code description should be updated to reflect that it is for the financing of the procurement/contracting process, not for payment.

Since "financing" might not be correctly interpreted by non-native English speakers, we should expand on what this concept means more specifically, e.g. with a brief example.

### PR DESCRIPTION
**Pull request by yolile**
Closes #937 #438

### CODE REVIEW

File: docs/history/changelog.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#998](hhttps://github.com/open-contracting/standard/pull/998) Add 'NAICS', 'PSC' and 'HS' codes.
   * [#967](https://github.com/open-contracting/standard/pull/967) Remove the discouragement of the 'UNSPSC' code.
 
+* `milestoneType.csv`:
```
Comment (jpmckinney) [inline docs/history/changelog.md:27]: Remove newline above this line.

File: docs/history/changelog.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ Per the [normative and non-normative content and changes policy](https://docs.go
...
+  * [#1000](https://github.com/open-contracting/standard/pull/1000) Update 'financing' description and add 'payment' code.
```
Comment (jpmckinney) [inline docs/history/changelog.md]: To adopt a consistent style:

> Update description of 'financing' code, and add 'payment' code.

File: schema/codelists/milestoneType.csv [hunk 3]
```diff
@@ -5,4 +5,5 @@ engagement,Engagement milestones,"For engagement milestones, such as a public he
 assessment,Assessment milestones,"For assessment and adjudication milestones, such as the meeting date of a committee."
 delivery,Delivery milestones,"For delivery milestones, such as the date when a good or service is to be provided."
 reporting,Reporting milestones,"For reporting milestones, such as when key reports are to be provided."
-financing,Financing milestones,"For events such as planned payments, or equity transfers in public private partnership projects."
+financing,Financing milestones,"For events related to the funding of the contracting process, such as planned payments or equity transfers in public private partnership projects."
```
Comment (jpmckinney) [inline schema/codelists/milestoneType.csv]: To address discussion in https://github.com/open-contracting/standard/issues/438#issuecomment-554032553:

> For events related to the funding of the contracting process. In public-private partnerships, such events might include, for example, planned payments and equity transfers.

File: schema/codelists/milestoneType.csv [hunk 4]
```diff
@@ -5,4 +5,5 @@ engagement,Engagement milestones,"For engagement milestones, such as a public he
...
+payment,Payment milestones,"For payment milestones, such as the date when a payment will be made to a supplier. When the ´met´ status is reached by this milestone, a transaction with information about the payment should be added."
```
Comment (jpmckinney) [inline schema/codelists/milestoneType.csv]: will -> is expected to

We can't add new rules ("should") in 1.1.5, so rephrase as:

> If the `status` of a contract's payment milestone is 'met', then the contract ought to have an entry for the payment under its `implementation.transactions`.

File: docs/history/changelog.md (hunk 1 above)
Comment (yolile) [reply docs/history/changelog.md:27]: done

File: docs/history/changelog.md (hunk 2 above)
Comment (yolile) [reply docs/history/changelog.md]: done

File: schema/codelists/milestoneType.csv (hunk 4 above)
Comment (yolile) [reply schema/codelists/milestoneType.csv]: done

File: docs/history/changelog.md (hunk 1 above)
Comment (jpmckinney) [reply docs/history/changelog.md:27]: Hmm, you removed the one after, instead of before :)

File: schema/codelists/milestoneType.csv [hunk 5]
```diff
@@ -5,4 +5,5 @@ engagement,Engagement milestones,"For engagement milestones, such as a public he
 assessment,Assessment milestones,"For assessment and adjudication milestones, such as the meeting date of a committee."
 delivery,Delivery milestones,"For delivery milestones, such as the date when a good or service is to be provided."
 reporting,Reporting milestones,"For reporting milestones, such as when key reports are to be provided."
-financing,Financing milestones,"For events such as planned payments, or equity transfers in public private partnership projects."
+financing,Financing milestones,"For events related to the funding of the contracting process. In public-private partnerships, such events might include, for example, planned payments and equity transfers."
+payment,Payment milestones,"For payment milestones, such as the date when a payment is expected to be made to a supplier. If the ´status´ of a contract's payment milestone is 'met', then the contract ought to have an entry for the payment under its ´implementation.transactions´."
```
Comment (jpmckinney) [inline schema/codelists/milestoneType.csv]: Use backticks (\`), not ´

File: docs/history/changelog.md (hunk 1 above)
Comment (yolile) [reply docs/history/changelog.md:27]: haha you're rigth! sorry! done!

**Comment by jpmckinney** [comment]
🎉 1000!

**Pull request merged**